apps/src/lab2/responseValidators.ts

```diff
@@ -89,8 +89,8 @@ export const SourceResponseValidator: ResponseValidator<
 export const LevelPropertiesValidator: ResponseValidator<
   LevelProperties
 > = response => {
-  if (!response.appName) {
-    throwMissingFieldError('appName');
+  if (Array.isArray(response) || !response.appName) {
```

Comment:
Could you throw a comment here around why the new Array condition is here (or throw a different error)? The current condition aligns so well with the error being thrown, but the new one I think might have future readers like 🤔

Reply:
Updated! Split into two separate checks and errors